react/react dependency installer/installer/e4e1b2e6-26a1-4b3b-b472-4464814fa69a.json

```diff
@@ -0,0 +1,55 @@
+{
+  "implementations": {
+    "es6-node": {
+      "inputFunctions": {
+        "045a62ad-7340-479a-b544-f10d56f16597": "const { exec } = require( \"child_process\" );\nconst { dependencies } = params;\nif (data.start !== false ){\n  const command = \"cd \" + data.projectPath + \" && \" + \"npm install \" + params.dependencies.join(' ') + \" react-router-dom\";\n  outputs.stdout( {\"stdout\" : command}, tag );\n  if (state.running){\n    outputs.bounced( {\"bounced\" : data}, tag);\n  } else {\n    state.running = true;\n    exec(command, (error, stdout, stderr) => {\n      if (error) {\n        outputs.error( {\"error\" : `${error}`}, tag);\n      } else {\n        outputs.stdout({\"stdout\" : stdout} || {\"stdout\" : \"\"}, tag);\n        outputs.stderr({\"stderr\" : stderr} || {\"stderr\" : \"\"}, tag);\n        outputs.success(true,tag);\n      }\n      state.running = false;\n    });\n  }\n}"
+      }
+    }
+  },
+  "interface": {
+    "inputs": {
+      "93078915-107a-4854-baaf-e2cdd4c06760": {
+        "type": "string[]",
+        "name": "dependencies",
```

Comment:
Ths internal code node should receive all the required parameters in one port (parent port can sync them together for it) in order to avoid issues due to asynchronous signals.